src/fides/api/models/privacy_notice.py

```diff
@@ -119,6 +119,7 @@ class UserConsentPreference(Enum):
         ("sk", "sk"),  # slovakia
         ("fi", "fi"),  # finland
         ("se", "se"),  # sweden
+        ("gb", "gb"),  # great britain
```

Comment:
thanks @pattisdr ! I think you are right, I wasn't sure about making this a default for new deployments or anything either in light of the new changes coming soon... Similar to Canada (which we should probably look to include as well across most of these). Are you ok with the manual management for this additional location or do you think you may prefer something else?

Reply:
Manual management seems fine here, in our multitranslation release, "gb" will automatically be there on new out of the box experiences.  

I haven't seen "ca" as something we're adding shipped out of the box yet -

Reply:
You could add gb to the privacy notice yaml file that we ship with out of the box too in fides potentially, that would cause gb to show up on the notices for new customers anyway in the interim. Also not sure if it is worth it though.